server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -692,6 +692,7 @@ private DivTag renderQuestion(
       Request request) {
     DivTag ret =
         div()
+            .withData("testid", "question-admin-name-" + questionDefinition.getName())
```

Comment:
This puts the testid on each question container that has been added onto a block. Otherwise it's difficult to correctly target buttons when there are multiple questions on a block.